lib/rock_rms/resources/refund_reason.rb

```diff
@@ -0,0 +1,39 @@
+module RockRMS
+  class Client
+    module RefundReason
+      def list_refund_reasons(options = {})
+
+        #
+        # assumes the DefinedType of "Refund Reason" is 37
+        #
+        options = {'$filter' => "DefinedTypeId eq 37"}
+        Response::DefinedValue.format(get(defined_values_path, options))
+      end
+
+      def create_refund_reason(value:, description:, order: nil)
+
+        order ||= Random.rand(1000) + 1000
+
+        options = {
+          'Description' => description,
+          'Value' => value,
+          'Order' => order,
```

Comment:
Why does `Order` matter?

Reply:
I couldn't create a DefinedValue in Rock without an Order value, and I assume they should be unique, so I'm trying to fill in some useless, but unique, integer